### ISSUE TITLE
Doc source links broken

### ISSUE THREAD

**Issue by ochrons** [akka/akka#24316]
The documentation source links on pages like https://doc.akka.io/docs/akka/2.5.5/scala/stream/stream-cookbook.html are broken. It should point to https://github.com/akka/akka/blob/master/akka-docs/src/main/paradox/stream/stream-cookbook.md but points to https://github.com/akka/akka/tree/master/akka-docs/src/main/paradox/scala/stream/stream-cookbook.md instead

**Comment by johanandren** [comment]
The problem is that master changed after those docs were published, with current the link works, we wouldn't want to point the link to a tag or branch either as it is basically there to make PR:ing small doc fixes easier.

If you look at the current version of the Akka docs the link is correct: https://doc.akka.io/docs/akka/2.5.9/stream/stream-cookbook.html?language=scala

**Comment by ochrons** [comment]
Yeah, that's what I also asked on akka/akka Gitter on how the old docs are maintained when there are fixes that affect old versions as well. Maybe have a separate version tag pointing to the most recent doc for that particular version?

**Comment by johanandren** [comment]
Otoh current will pretty much always work, and you'd anyway have to also go look at current to fix docs related things, because it may already be fixed in master.

**Comment by raboof** [comment]
Perhaps we should only have the link to the sources of the docs on 'current' (and perhaps 'snapshot') version of the docs?

For older versions we should link to the 'current' version of that page (#23869)

**Comment by raboof** [comment]
This is now fixed in more recent versions of the docs (e.g. https://doc.akka.io/docs/akka/2.5.22/stream/stream-cookbook.html?language=scala)

**Comment by jrudolph** [comment]
Actually, it now always points to the tag version of the page which is good for consistency but which you cannot edit right away. I filed https://github.com/lightbend/paradox/issues/325 to allow linking always to master.
